### ISSUE TITLE
Better documentation for the Epmax and Eemax parameters

### ISSUE THREAD

**Issue by luca-giunti** [zblz/naima#207]
While studying candidate PeVatron sources, the high-energy part of the electron/proton distributions are extremely relevant. For a long time, I used naima without modifying the default Epmax=10 PeV and Eemax=510 TeV parameters, just because I didn't know they existed, thus strongly biasing my results. 

A more explicit handling of this parameters (also the Epmin/Eemin) would be good. For example, not defining any default Epmax/Eemax would force the user to make a choice, adapted case-by-case. Alternatively, the default values should be well documented.

Thanks, cheers!

**Comment by zhenjieW** [comment]
Hi, the Epmin/Eemin is optional, it is a variable.

**Comment by luca-giunti** [comment]
Yes, the problem is that if you don't set it properly all your results at high energy may be wrong. It is OK to leave it optional, but I would document it better if possible
